samza-core/src/main/java/org/apache/samza/task/DefaultTaskExecutorFactory.java

```diff
@@ -50,9 +50,8 @@ public ExecutorService getTaskExecutor(Config config) {
    * {@inheritDoc}
    *
    * The choice of thread pool is determined based on the following logic
-   *    If job.operator.thread.pool.enabled,
-   *     a. Use {@link #getTaskExecutor(Config)} if job.container.thread.pool.size &gt; 1
-   *     b. Use default single threaded pool otherwise
+   *     1. Use {@link #getTaskExecutor(Config)} if job.container.thread.pool.size &gt; 1
```

Comment:
Nit: This change seems unrelated and doesn't affect build. Do we need this with this PR?

Reply:
Yes. but I didn't want to open a separate PR for this one line java doc change.